src/AI/agents/agents/services/mcp/mcp_client.py

```
@@ -0,0 +1,410 @@
+
+async def check_mcp_server_startup(server_url: str = None, expected_version: str = None):
+    """
+    Check MCP server status and version at startup.
+    
+    Args:
+        server_url: MCP server URL. If None, uses config default.
+        expected_version: Expected version string. If None, uses config default.
+        
+    Exits the application with code 1 if MCP server check fails.
+    """
+    import os
```

Ruff (0.14.8)

Redefinition of unused `os` from line 6

Remove definition: `os`

(F811)